README.md

```diff
@@ -4,14 +4,13 @@ LighterPack helps you track the gear you bring on adventures.
 
 How to run Lighterpack
 -----------
-
-1. Install node.js, npm and mongo
-2. ```$ git clone https://github.com/galenmaly/lighterpack.git```
-3. Install dependancies ```$ npm install```
-4. start mongo ```$ mongod```
-5. Start app ```$ npm start```
-6. go to http://localhost:8080
+1. Install [Node.js](https://nodejs.org/en), NPM and [MongoDB](https://www.mongodb.com)
+2. Clone this repository: ```$ git clone https://github.com/galenmaly/lighterpack.git```
+3. Install the dependencies: ```$ npm install```
+4. Start MongoDB ```$ mongod```
+5. Start application ```$ npm start```
```

Comment:
Only need single tick mark for inline code, tripple are for use with a code block where you can specify the syntax to highlight. 

```suggestion
2. Clone this repository: `$ git clone https://github.com/galenmaly/lighterpack.git`
3. Install the dependencies: `$ npm install`
4. Start MongoDB `$ mongod`
5. Start application `$ npm start`
```